### PR TITLE
docs(qa,smoke tests): Add AF3 Smoke Test

### PR DESCRIPTION
**Pull request by podarok**
Fixes: https://openy.atlassian.net/browse/MAINTAIN-46
Origin reference: https://docs.google.com/spreadsheets/d/1yLUkMgJKK94hABy107_V-1AcJbRSSEf2s4wsQto1wfI/edit#gid=0&range=A74

Extends: https://github.com/ymcatwincities/openy/wiki/Open-Y-Smoke-Tests-Index

### CODE REVIEW

File: openy_af_vue_app/SMOKE_TESTS.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+1. Log in as admin
+2. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+3. Click edit on Activity Finder paragraph
+4. Check the ""Hide Program Categories"" checkbox
+5. Save node and check the Activity finder
+
+
+### Expected Results
+
+1. The select location step sould be skipped
```
Comment (hamrant) [inline openy_af_vue_app/SMOKE_TESTS.md]: Should be `select program step` instead of `select location step`

File: openy_af_vue_app/SMOKE_TESTS.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+- Expandable filters in different groups (see test case below)
+2. By clicking on any filters the results should be updated with counting of selected filter
+3. On the top of results you should be able to sort results: 
+- Sort by Title (A-Z)
+- Sort by Title (Z-A)
+- Sort by Location (A-Z)
+- Sort by Location (Z-A)
+- Sort by Activity (A-Z)
+- Sort by Activity (Z-A)
+4. All manipulations on the page happens without reloading the page
```
Comment (hamrant) [inline openy_af_vue_app/SMOKE_TESTS.md:226]: Let's add a step with checking browser console for errors:

```
5. Open browser console (press F12 on keyboard). Check there are no errors related to AF3 in logs.
```

**Review by hamrant — CHANGES_REQUESTED** [review]

File: openy_af_vue_app/SMOKE_TESTS.md (hunk 2 above)
Comment (podarok) [reply openy_af_vue_app/SMOKE_TESTS.md:226]: done https://github.com/ymcatwincities/openy_activity_finder/pull/80/commits/65dd9c2829cc8ecb4a6e15bbc2e9edf91bee5059

File: openy_af_vue_app/SMOKE_TESTS.md (hunk 1 above)
Comment (podarok) [reply openy_af_vue_app/SMOKE_TESTS.md]: fixed, tnx

**Review by hamrant — APPROVED** [review]

**Pull request merged**